src/Screens/Main/BuddyList/Button.tsx

```diff
@@ -22,9 +22,16 @@ const Button = ({ style, buddyId, name, onPress, ...viewProps }: Props) => {
   return (
     <Card style={[styles.button, style]} {...viewProps}>
       <RN.TouchableOpacity style={styles.content} onPress={onPressBuddy}>
-        <RN.Text style={styles.nameText}>{name}</RN.Text>
+        <RN.Text style={styles.nameText} ellipsizeMode="tail" numberOfLines={1}>
+          {name}
+        </RN.Text>
         <RN.View style={[styles.blob, { backgroundColor: color }]}>
-          {hasNewMessages ? <RN.View style={styles.newMessage} /> : null}
+          {hasNewMessages ? (
+            <RN.View
+              style={styles.newMessage}
+              testID={'Main.BuddyList.Button.unseenDot'}
```

Comment:
testids seem to have all lowercase in other places... how should it be?

Reply:
True, should be changed, its disturbing.. 😁